### ISSUE TITLE
ActuonixL1250Motor is not being detected

### ISSUE THREAD

**Issue by sheenamelwani52** [ev3dev/ev3dev#1473]
I am trying to use ev3 actuonix L1250 linear motor with feedback control using ActuonixL1250Motor Class but it not being detected on EV3. 

When I run dmesg | tail, it detects as an ev3 Large Motor instead of linear. 
![dmesg tail output](https://user-images.githubusercontent.com/51267238/108890225-9916e900-760d-11eb-93c8-834a957bcb99.JPG)

I also tried to run it with LargeMotor and Motor class but in that case, position and position_sp do not provide proper values. It remains 0 even after the motor is fully contracted or extended. And also count_per_m will not work since it is not detected as a linear motor.

**Code:**

```python
from ev3dev2.motor import Motor, ActuonixL1250Motor, LargeMotor, OUTPUT_B, SpeedPercent

def main():
    m = ActuonixL1250Motor(OUTPUT_B)
    reset_console()
    set_cursor(OFF)
    set_font('Lat15-Terminus24x12')
    print ("Position {}".format(m.position))
    print ("Tachos {}".format(m.position_sp))
    print ("count/rot {}".format(m.count_per_rot))
    #print ("count/m {}".format(m.count_per_m))
    
    m.on_for_seconds(SpeedPercent(-75), 1)

    time.sleep(5)
   
    reset_console()
    set_cursor(OFF)
    set_font('Lat15-Terminus24x12')

    print ("Position{}".format(m.position))
    print ("Tachos {}".format(m.position_sp))
    print ("count/rot {}".format(m.count_per_rot))
   #print ("count/m {}".format(m.count_per_m))
    time.sleep(5)
```

**Comment by dlech** [comment]
These motors can't be automatically detected. You will need to use a [LegoPort](https://ev3dev-lang.readthedocs.io/projects/python-ev3dev/en/stable/ports.html#ev3dev2.port.LegoPort) object and call `set_device = 'act-l12-ev3-50'` to get the correct driver. You will probably need a delay after this before creating the `ActuonixL1250Motor` object.

**Comment by dlech** [comment]
Alternately, you can use the device browser on the EV3 to manually configure the port.

**Comment by sheenamelwani52** [comment]
Thank you for your quick response. I did the following edits as suggested

```python
from ev3dev2 import list_devices
from ev3dev2.port import LegoPort

  p1 = LegoPort(OUTPUT_B)
  p1.set_device = 'act-l12-ev3-50'
  time.sleep(2)
```

However, it is still not resolved, my visual studio output
![VS output](https://user-images.githubusercontent.com/51267238/108895139-7e477300-7613-11eb-9aff-9c066494ee96.JPG)

In dmesg | tail, I can see now it is detecting but also automatically removing?
![image](https://user-images.githubusercontent.com/51267238/108895302-b3ec5c00-7613-11eb-8aef-309b6c6e9e66.png)

**Comment by dlech** [comment]
Might need to take the port out of `auto`

    p1.mode = 'tacho-motor'

**Comment by sheenamelwani52** [comment]
Thanks, I tried that already but still gives the same error. By the way, is this motor supported on Brickpi3?

**Comment by dlech** [comment]
BrickPi3 treats everything as an NXT motor IIRC.

**Comment by sheenamelwani52** [comment]
Yes, that's right but can I configure the port by the same method as you have mentioned for ev3?
